### PR TITLE
Adapt text and bootstrap size in blog sample data

### PR DESCRIPTION
**Pull request by chmst**
Pull Request for Issue #33832 .

### Summary of Changes
Change text to "To totally change the look of your site you will probably want to install a new template. **Go to System, select Install - Extensions from the list and the extension installer will open**. There are many free" 

Bootstrap size of module site information is set to 0, to fit recently made changes in dashboard modules. #33045 et alii

### Testing Instructions
Code review or install blog sample data, 
Front end : Log in and and check the text "Working on your Site"
Backend:  Check the with of the module site information

### Actual result BEFORE applying this Pull Request
Module site information has boostrap size 6 and has only half the width of the column
Text see  #33832 .

### Expected result AFTER applying this Pull Request
Module site information has same size as other modules
Text is correct for J4

### CODE REVIEW

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 824afc92293a4faf5fd77eb3829e1b6047e9648d<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33866">issues.joomla.org/tracker/joomla-cms/33866</a>.</sub>

**Comment by YatharthVyas** [comment]
I have tested this item :white_check_mark: successfully on 824afc92293a4faf5fd77eb3829e1b6047e9648d<br /><br />Successfully verified both the updated text and the module's bootstrap size on a Nightly Build.

Working on your site

image

Site Information (Statistics) Module

image

<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33866">issues.joomla.org/tracker/joomla-cms/33866</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33866">issues.joomla.org/tracker/joomla-cms/33866</a>.</sub>

File: plugins/sampledata/blog/blog.php [hunk 1]
```diff
@@ -1713,7 +1713,7 @@ public function onAjaxSampledataApplyStep3()
 					'cache_time'     => 900,
 					'cachemode'      => 'static',
 					'module_tag'     => 'div',
-					'bootstrap_size' => 6,
+					'bootstrap_size' => 0,
```
Comment (richard67) [inline plugins/sampledata/blog/blog.php:1716]: Is 0 right? Other modules got 12 in the PR #33045 which you've mentioned.

Comment (richard67) [reply plugins/sampledata/blog/blog.php:1716]: @brianteeman Maybe you can advise me if zero is the right value here or if it should be 12?

Comment (richard67) [reply plugins/sampledata/blog/blog.php:1716]: Assuming zero means default and will result in 12, I will set this PR RTC.

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33866">issues.joomla.org/tracker/joomla-cms/33866</a>.</sub>

**Comment by HLeithner** [comment]
thanks

**Pull request merged**